src/components/AppProvider/AppProvider.tsx

```diff
@@ -32,8 +32,8 @@ interface State {
 }
 
 export interface AppProviderProps extends AppBridgeOptions {
-  /** A locale object or array of locale objects that overrides default translations */
-  i18n: TranslationDictionary | TranslationDictionary[];
+  /** A locale object or array of locale objects that overrides default translations. If specifying an array then your fallback language dictionaries should come first, followed by your primary language dictionary */
+  i18n: ConstructorParameters<typeof I18n>[0];
```

Comment:
I wonder how this will appear in the style guide 🤔 And hopefully not `ConstructorParameters<typeof I18n>[0]` 😬

Reply:
Looks like it outputs the correct type 👍